docs/src/man/comparisons.md

```diff
@@ -0,0 +1,27 @@
+The table compares the main functions of DataFrames with R dplyr and Stata
+| DataFrames       | dplyr | Stata|
+|:------------|:------------|:------------|
+|`combine(df, :x => mean)`  | `summarize(mean(x))`    | `collapse (mean) x `|
+|`transform(df, :x => mean)`   | `mutate(mean(x))`    | `egen x_mean = mean(x)`|
+|`select(df, :x, :y)`   | `select(df, x, y)`  | `keep x y` |
+|`sort(df, :x)`   | `arrange(df, x)`    | `sort x`|
+|`filter(:x => x -> x >= 1, df)`   | `filter(df, x >= 1)`  | `keep if x >= 1` |
```

Comment:
```suggestion
|`filter(:x => >=(1), df)`   | `filter(df, x >= 1)`  | `keep if x >= 1` |
```

Reply:
I understand that it is a shorthand but it seems a bit harder to understand, and less generalizable, so I'm not sure I want to change it

Reply:
OK